Sources/SQLKit/SQLRowDecoder.swift

```diff
@@ -1,8 +1,28 @@
-struct SQLRowDecoder {
-    func decode<T>(_ type: T.Type, from row: SQLRow, prefix: String? = nil) throws -> T
+public struct SQLRowDecoder {
+    public var keyPrefix: String? = nil
+    public var keyDecodingStrategy: KeyDecodingStrategy = .useDefaultKeys
+
+    func decode<T>(_ type: T.Type, from row: SQLRow) throws -> T
         where T: Decodable
     {
-        return try T.init(from: _Decoder(prefix: prefix, row: row))
+        return try T.init(from: _Decoder(row: row, options: options))
+    }
+
+    public enum KeyDecodingStrategy {
+        case useDefaultKeys
+        case convertFromSnakeCase
```

Comment:
Some comments here would be nice. I assume this means "convert from snake case to camel case", right?